### DISCUSSION TITLE
Why add condition 81(TFCond_BlastJumping) to boss when superjump?

### DISCUSSION THREAD

**Discussion by M60TM** [Batfoxkid/Freak-Fortress-2-Rewrite#148] [Q&A] [answered]
for `mod mini-crit airborne`? or support `rocketjump attackrate bonus`?

**Comment by Batfoxkid** [comment, marked as answer]
For this exact reasons, gives a benefit to weapons such as the Reserve Shooter, especially in cases where a boss has too much knockback resistance to knock into the air or needing to hit with the Direct Hit.

**Reply by M60TM** [reply]
I see, but it wasn't enough to do that. I'll pull request soon.
